charts/jaeger/README.md

```diff
@@ -68,18 +68,24 @@ data:
 
 ## Installing the Chart
 
+Add the Jaeger Tracing Helm repository:
+
+```console
+$ helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
```

Comment:
Visiting https://jaegertracing.github.io/helm-charts/ shows 404. Should there be some landing page, maybe pointing back to this repo at minimum?

Reply:
A Helm Charts repo just needs an index.yaml that point to valid chart resources, and many don't have landing pages if you visit them (for example, the helm/charts stable repo landing page is just XML https://kubernetes-charts.storage.googleapis.com/). But it might be nice to add a friendly page 😄  I've seen some good ones, and gh-pages hosting method makes this very easy too

Reply:
Let's look at that in a separate PR. Here's an example that's kind of nice and also easy https://charts.fluxcd.io/